frontend/packages/process-editor/src/types/BpmnDetails.ts

```diff
@@ -8,4 +8,8 @@ export interface BpmnDetails {
   taskType: BpmnTaskType | null;
   type: BpmnTypeEnum;
   element?: Element;
+
+  metadata?: {
+    justAdded?: boolean;
```

Comment:
Ref. previous comment. Let's try to create a generic and reusable concept to know when to show RecommendedNextAction. 😊